pkg/controller/statefulset/statefulset_controller_test.go

```diff
@@ -823,7 +823,7 @@ func NewStatefulSetController(
 			},
 			DeleteFunc: ssc.enqueueStatefulSet,
 		},
-		statefulSetResyncPeriod,
+		30*time.Second,
```

Comment:
we should define const, instead of using the value directly.

Reply:
ok, will  define const in this file instaead of statefulset_controller